packages/semantic-ui-theme/semantic/src/themes/sharegate/modules/dropdown.overrides

```diff
@@ -215,3 +292,99 @@
     height: 2rem;
     max-height: 2rem;
 }
+
+/* Content | With Icon */
+.ui.selection.dropdown .menu > .item {
```

Comment:
Are you sure you want to target only `.ui.selection` dropdown? Since items feature are supported for all kind of dropdown it seems limitative. For example, would it also work with an inline dropdown?

Same comments for the other selectors below